src/interaction/sessionrecording.cpp

```diff
@@ -1713,7 +1713,10 @@ std::vector<std::string> SessionRecording::playbackList() const {
     std::vector<std::string> allInputFiles = currentDir.readFiles();
     for (const std::string& f : allInputFiles) {
         // Remove path and keep only the filename
-        fileList.push_back(f.substr(path.length() + 1, f.length() - path.length() - 1));
+        const std::string filename = f.substr(path.length() + 1, f.length() - path.length() - 1);
+        if (filename.rfind(".", 0) != 0) { //Don't add hidden files
```

Comment:
a) Why the rfind?  I don't think this will work if you have a hidden file that has a file extension since it will find the last . in the file name that is the separator
b) This is a unix-way of finding hidden files.  On `WIN32` you'll need to instead check for:
```
DWORD attributes = GetFileAttributes(filename);
bool isHidden = attributes & FILE_ATTRIBUTE_HIDDEN;
```